src/update_sdk.cmd

```diff
@@ -0,0 +1,43 @@
+REM This script replaces the SDK folder with the latest bits from the Azure SDK repository.
+REM It removes some files we do not need.  It currently targets the develop branch until fixes are
+REM merged into master.
+
```

Comment:
Any thoughts on a nice cross platform (Windows, OS X, Linux) way to do this? The previous script was based on bash, which wouldn't run on Windows, however the .cmd file can't run on OS X and Linux.

Just something to think about, I imagine you'll mostly be syncing the SDK from now on.